systemtests/aci_test.go

```diff
@@ -386,11 +393,188 @@ func (s *systemtestSuite) TestACIProfile(c *C) {
 		c.Assert(cA2.checkPingFailureWithCount(cB2.eth0.ip, 5), IsNil)
 
 		c.Assert(s.removeContainers([]*container{cA1, cB1, cA2, cB2}), IsNil)
-		c.Assert(s.cli.EndpointGroupDelete("aciTenant", "epgA"), IsNil)
-		c.Assert(s.cli.EndpointGroupDelete("aciTenant", "epgB"), IsNil)
-		c.Assert(s.cli.RuleDelete("aciTenant", "policyAB", "2"), IsNil)
-		c.Assert(s.cli.RuleDelete("aciTenant", "policyAB", "3"), IsNil)
-		c.Assert(s.cli.PolicyDelete("aciTenant", "policyAB"), IsNil)
-		c.Assert(s.cli.NetworkDelete("aciTenant", "aciNet"), IsNil)
+		c.Assert(s.cli.EndpointGroupDelete(s.acinfo.Tenant, "epgA"), IsNil)
+		c.Assert(s.cli.EndpointGroupDelete(s.acinfo.Tenant, "epgB"), IsNil)
+		c.Assert(s.cli.RuleDelete(s.acinfo.Tenant, "policyAB", "2"), IsNil)
+		c.Assert(s.cli.RuleDelete(s.acinfo.Tenant, "policyAB", "3"), IsNil)
+		c.Assert(s.cli.PolicyDelete(s.acinfo.Tenant, "policyAB"), IsNil)
+		c.Assert(s.cli.NetworkDelete(s.acinfo.Tenant, s.acinfo.Network), IsNil)
+	}
+}
+
+func (s *systemtestSuite) TestACIGW_Restart(c *C) {
```

Comment:
Couple of high level comments on this test:

1. Its good to write tests such that it can run on any number of nodes. Not just a single node. This way we can reuse the tests on different kinds of setups. Please see non-ACI tests for an example
2. We should take number of iteration as an argument and run the tests multiple times. please see other tests for an example.
3. We should also take number of containers as an argument too.
4. If there are different things being tested, like connectivity and policy. Its generally best to break that up into two separate tests.
5. Its best to separate the initialization (i.e, part where you setup network/epg/policy/rule ) and test run (i.e. where you are running the containers and checking ping)